<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProgramJava</class>
 <widget class="QWidget" name="NewProgramJava">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Java Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="combo_version">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_main_class">
       <property name="text">
        <string>Main Class:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="edit_main_class"/>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="edit_jar_file"/>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="combo_start_mode">
       <item>
        <property name="text">
         <string>Main Class</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JAR File</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Mode:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_jar_file">
       <property name="text">
        <string>JAR File:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QLabel" name="label_jar_file_help">
       <property name="text">
        <string>In JAR mode [-jar] the specified Java archive will be executed.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLabel" name="label_main_class_help">
       <property name="text">
        <string>In Class mode the specified Java class will be executed.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QListWidget" name="list_arguments"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arguments:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="button_add_argument">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="button_remove_argument">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>This list of arguments will be passed to the main() method.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="button_down_argument">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="button_up_argument">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
